app/namespace.go

```diff
@@ -1535,6 +1536,12 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 					s3BucketFlagRequired,
 					RequestIDFlag,
 					kmsArnFlag,
+					&cli.StringFlag{
```

Comment:
nit: should this be a variable to follow original design pattern

Reply:
I'll make it a variable before merging